src/site/_includes/components/updates/index.scss

```
@@ -0,0 +1,6 @@
+@use '@moduk/frontend/src/css' as moduk;
+
+.guidance-updates {
+  background-color: moduk.govuk-colour('light-grey');
+  border-bottom: 1px solid moduk.govuk-colour('mid-grey');
```

Might be good to use ` $govuk-border-colour` for this one:

```suggestion
  border-bottom: 1px solid moduk.$govuk-border-colour ;
```